revolv/donor/views.py

```diff
@@ -30,3 +34,78 @@ def get_context_data(self, **kwargs):
         context["donor_has_no_donated_projects"] = Project.objects.donated_projects(self.user_profile).count() == 0
 
         return context
+
+class AccountingView(UserDataMixin, TemplateView):
+    """
+    View of the accounting page.
+    """
+    template_name = 'donor/accounting.html'
+
+    def get_context_data(self, **kwargs):
+        context = super(AccountingView, self).get_context_data(**kwargs)
+
+        context['project_names'] = [p.title for p in Project.objects.all()]
+        context['admin_repayments'] = AdminRepayment.objects.repayments()
+        context['admin_reinvestments'] = AdminReinvestment.objects.reinvestments()
+        context['donations'] = Payment.objects.donations()
+        context['jsonurl'] = unicode(str(reverse('donor:accountingPartial')))
+
+        context['clean_project_names'] = {}
```

Comment:
I'm still not sure why you need clean project names to be class names in html. Can't you just called the classes `project-{{project.pk}}`?